Arduino_Sub_DAQ/Arduino_Sub_DAQ.ino

```diff
@@ -0,0 +1,149 @@
+/*
+ *  Linear Potentiometer Microcontroller Code
+ * 
+ *  By: Basel Jazzar - Suspension Lead
+ *  8/25/22
+ *  
+ *  Dallas Formula Racing - Formula SAE
+ *  
+ */
+
+
+#include <SPI.h>
+#include <SD.h>
+File myFile;
+
+bool hasntClose = true;
+String file_name;
+int file_name_num = 1;
+int sensorValue1 = -1;
+int sensorValue2 = -1;
+int sensorValue3 = -1;
+int sensorValue4 = -1;
+
+//A2, A3 are front, A0, A1 are rear
+int anaPins[] = {A0, A1, A2, A3};
+
+int anaVals[] = {-1, -1, -1, -1};
+int rstPin = 6;
+
+void setup() {
+
+  Serial.begin(9600);
+  hasntClose = true;
+  
+  while(!Serial){
+    ;
+  }
+  
+  Serial.print("Initializing SD card...");
+  
+  while (!SD.begin(10)) {
+    Serial.println("Initialization failed");
+  }
+  Serial.println("Initialization done");
+
+  file_name = "data_" + String(file_name_num) + ".csv";
```

Comment:
The air pressure sensor is close to being usable. Consider a more specific filename in case the same SD card is used to log the data.
- [ ] Change the filename